### ISSUE TITLE
Can't compile against PG16 for latest released 1.5.0 - cut a new release

### ISSUE THREAD

**Issue by robe2** [pramsey/pgsql-http#161]
I'm getting this error when I try to compile against PG16 beta2.
I'll provide a patch once I figure out what's wrong

```
http.c
http.c:597:1: error: static declaration of 'get_extension_schema' follows non-static declaration
 get_extension_schema(Oid ext_oid)
 ^~~~~~~~~~~~~~~~~~~~
In file included from http.c:54:
C:/MING64~1/projects/POSTGR~1/rel/PG16W6~2/include/server/commands/extension.h:50:12: note: previous declaration of 'get_extension_schema' was here
 extern Oid get_extension_schema(Oid ext_oid);
            ^~~~~~~~~~~~~~~~~~~~
http.c: In function 'http_request':
http.c:1146:9: warning: declaration of 'content_type' shadows a previous local [-Wshadow=compatible-local]
   char *content_type;
         ^~~~~~~~~~~~
http.c:1004:8: note: shadowed declaration is here
  char *content_type = NULL;
        ^~~~~~~~~~~~
make: *** [<builtin>: http.o] Error 1

```

**Comment by robe2** [comment]
False alarm.  I was building against 1.5.0 release.  I am only getting a warning about the:

```
http.c: In function 'http_request':
http.c:1146:9: warning: declaration of 'content_type' shadows a previous local [-Wshadow=compatible-local]
   char *content_type;
         ^~~~~~~~~~~~
http.c:1004:8: note: shadowed declaration is here
  char *content_type = NULL;
```

you should probably fix that. 

Anyway chance you can make a release that works against PG16?  I'd rather not be shipping untagged versions.

**Comment by robe2** [comment]
@pramsey So can we have a 1.6.0 tag soon so I can build a tagged version for PG16 pretty please :)

**Comment by pramsey** [comment]
Ha ha ha, you funny! 

I just released 1.5... I guess numbers are cheap.

**Comment by robe2** [comment]
You released 1.5 more than a year ago April 2022

**Comment by pramsey** [comment]
Done 

https://github.com/pramsey/pgsql-http/releases/tag/v1.6.0
